code/go/0chain.net/conductor/conductrpc/server.go

```diff
@@ -392,6 +394,12 @@ func (s *Server) ShareOrSignsShares(soss *ShareOrSignsSharesEvent,
 	return
 }
 
+// magic block handler
+func (s *Server) MagicBlock(_ *struct{}, configFile *string) (err error) {
+	(*configFile) = s.magicBlock
```

Comment:
it doesn't make any sense, we can't pass entities by reference

Reply:
I am following the same convention as the other similar methods. What do you suggest doing?